app/views/home/index.html.erb

```diff
@@ -7,18 +7,13 @@
 
 		<!--Welcome section-->
         <section class="waypoint video-block fullscreen" data-hash="welcome">
-			<%= video_tag(
-				"home/woman-small.mov",
-				id: "womanvideo",
-				autoplay: "autoplay",
-				muted: "muted",
-				loop: "loop",
-				poster: "home/woman_front.jpg",
-				class: "background-video",
-				"x-webkit-airplay": true,
-				"webkit-playsinline": true,
-				preload: "auto"
-			) %>
+
+			<video id="womanvideo" autoplay="autoplay" muted="muted" loop="loop" poster    ="assets/home/woman_front.jpg" 
+			 	    class="background-video" x-webkit-airplay="true" webkit-playsinline="true" preload="auto">
+					<source data-src="assets/home/woman-small.mov" type="video/mp4">
+					<source data-src="assets/home/woman-small.ogm" type="video/ogm">
+			</video>
+
```

Comment:
Yea, QQ and Wechat browser, they both suffered. 
My mobile wechat browser is downloading the videos. 
The performance hurt is quite minimum, the video content will still be cached in the cloud front edge. Just the cloud front needs to send additional http request to server to validate the cached video stays the same.  If to use pipeline, this http request is omitted due to the hash the pipeline offers. This http request is ms level, and only applies to PC users.